generator/cli.ts

```diff
@@ -190,7 +207,7 @@ if (options.configFile) {
   options.shortName = pascalCase(options.title);
   options.date = getDate();
 
-  if (!options.author) {
+  if (!options.author && whitelabelPools.length == 0) {
```

Comment:
shouldn't we maintain the author at least? (could be helpful if whiteLabel have multiple contributors)